functional-tests/README.md

```diff
@@ -30,7 +30,7 @@ Prerequisities:
 docker run -p 10000:10000 -e "GITHUB_TOKEN_URL=https://auth.openshift.io/api/token?for=https://github.com" -e "OPENSHIFT_TOKEN_URL=https://sso.openshift.io/auth/realms/fabric8/broker/openshift-v3/token" -e "CHE_SERVER_URL=https://che.openshift.io" quay.io/openshiftio/almighty-che-starter:latest
 ```
 
-### Run tests via maven:
+### Run tests via maven
```

Comment:
Could you add a link to this readme to the [dev guide](https://github.com/redhat-developer/rh-che/blob/master/dev-guide.adoc)

Reply:
will do